### DISCUSSION TITLE
Corepulse - The Pimcore CMS / New Admin UI for Backend.

### DISCUSSION THREAD

**Discussion by starfruitjsc** [pimcore/pimcore#16549] [General]
Guys, we have publish bundle in Packagist so anyone can try our Pimcore CMS for your project

### What is Corepulse?
Corepulse is a CMS generator (not an UI bootstrap), it inherits all the PIM & DAM setup in Pimcore backend and make data management in an intuitive & user-friendly CMS.

**See Demo live** https://demok.goterest.com/cms

### How to Install?
Install Corepulse is as simple as any other bundle

- Get it in Packagist https://packagist.org/packages/corepulse/corepulse-bundle
- Installation tutorial https://coda.io/@starfruitjsc/corepulse/tutorials-4

![image](https://github.com/pimcore/pimcore/assets/41600972/ab0c7153-011e-43a6-8306-98eea1304ee1)

### Our Roadmap...
A long journey starts and we're trying to update almost everyday... You can follow our current Roadmap [here](https://coda.io/@starfruitjsc/corepulse) or my [Linkedin](https://www.linkedin.com/in/tr%E1%BB%8Bnh-t%C3%B9ng-anh-747361b3/) to stay tuned for new things.

If you have any feedback, pls don't hesitate to post it here so we can reply & fix it asap. 😁

**Comment by gitmini** [comment]
very very good !!!

**Comment by fashxp** [comment]
Hey folks, we also have some news regarding UI, have a look here: https://github.com/orgs/pimcore/discussions/17076
